Dockerfile.dev

```diff
@@ -3,10 +3,9 @@
 # especially useful for changes to dependencies or for final
 # verification of COSA code.
 #
-# This moves the build time from ~15 minutes to ~3-5 minutes.
 # Example: buildah bud -f Dockerfile.dev -t localhost/cosa-test .
 
-FROM quay.io/coreos-assembler/coreos-assembler:latest
```

Comment:
I think this is a fundamental part of the difference between `Dockerfile.dev` and `Dockerfile` though. With the previous suggestion to use `Dockerfile` instead, we should be able to just leave this one alone.